src/sage/knots/knotinfo.py

```diff
@@ -335,6 +335,45 @@ def knotinfo_bool(string):
         return False
     raise ValueError('%s is not a KnotInfo boolean')
 
+
+class SymmetryMutant(Enum):
+    r"""
+    Enum to specify the symmetry mutant link of the prime link listed in the
+    KnotInfo and LinkInfo databases. From the KnotInfo description page:
+
+        If a knot is viewed as the oriented diffeomorphism
+        class of an oriented pair, `K = (S_3, S_1)`, with `S_i`
+        diffeomorphic to `S^i`, there are four oriented knots
+        associated to any particular knot `K`. In addition to
+        `K` itself, there is the reverse, `K^r = (S_3, -S_1)`,
+        the concordance inverse, `-K = (-S_3, -S_1)`, and the
+        mirror image, `K^m = (-S_3, S_1)`.
+    """
+    itself = 's'
+    reverse = 'r'
+    concordance_inverse = 'mr'
+    mirror_image = 'm'
+    mixed = 'x' # to be used in connection with KnotInfoSeries
+    unknown = '?'
+
+    def __gt__(self, other):
+        r"""
+        Implement comparison of different items in order to have ``sorted`` work.
+
+        EXAMPLES::
+
+            sage: from sage.knots.knotinfo import SymmetryMutant
+            sage: sorted(SymmetryMutant)        # indirect doctest
+            [<SymmetryMutant.mixed: 'x'>,
+            <SymmetryMutant.itself: 's'>,
+            <SymmetryMutant.reverse: 'r'>,
+            <SymmetryMutant.concordance_inverse: 'mr'>,
+            <SymmetryMutant.mirror_image: 'm'>,
+            <SymmetryMutant.unknown: '?'>]
+        """
+        return self.value < other.value
```

Comment:
This is a `__gt__` method but the comparison is `lt`. Is this intentional?

Reply:
Yes! I want to have `itself` be the first of the proper items. I will add a comment in the code!